src/test/groovy/sdkman/specs/EnvCommandSpec.groovy

```diff
@@ -36,20 +43,50 @@ class EnvCommandSpec extends SdkmanEnvSpecification {
 		}
 
 		where:
-		sdkmanrc << ["grails=2.1.0\ngroovy=2.4.1", "grails=2.1.0\ngroovy=2.4.1\n"]
+		sdkrc << [
+			"grails=2.1.0\ngroovy=2.4.1",
+			"grails=2.1.0\ngroovy=2.4.1\n",
+			"  grails=2.1.0\ngroovy=2.4.1\n",
+			"grails=2.1.0	\ngroovy=2.4.1\n",
+			"grails=2.1.0\ngroovy = 2.4.1\n",
+		]
 	}
 
-	def "should issue an error if .sdkmanrc contains malformed candidate entries"() {
+	def "should issue an error if .sdkmanrc contains malformed candidate versions"() {
 		given:
-		new File(bash.workDir, '.sdkmanrc').text = "groovy 2.4.1"
+		new File(bash.workDir, ".sdkmanrc").text = "groovy 2.4.1"
 
 		when:
 		bash.execute("sdk env")
 
 		then:
 		verifyAll(bash) {
-			status > 0
+			status == 1
 			output.contains("Invalid candidate format!")
 		}
 	}
-}
+
+	def "should support blank lines, comments and inline comments"() {
+		given:
+		new FileTreeBuilder(candidatesDirectory).with {
+			"groovy" {
+				"2.4.1" {}
+			}
+		}
+
+		new File(bash.workDir, ".sdkmanrc").text = sdkrc
+
+		when:
+		bash.execute("sdk env")
+
+		then:
+		bash.output.contains("Using groovy version 2.4.1 in this shell.")
+
+		where:
+		sdkrc << [
```

Comment:
It is just naming, but let's call it `sdkmanrc` throughout.

Reply:
Consistency is important, I agree.